core/src/Select/Select.less

```diff
@@ -43,12 +33,15 @@
     }
 
     .chevron {
-      // border: 1px solid #404759;
```

Comment:
What's the reason for removing the border?

Reply:
@shrunyan  this was commented out code not in use, so  I cleaned it up